<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demande</class>
 <widget class="QWidget" name="Demande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LineLib">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineMarque">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Libellé</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Référence</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineRef">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Marque</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBut">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ajout_Panier">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter Au Panier</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categorie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Aucune Categorie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ordinateur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reseaux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Perepherique</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autre</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="subCategorie">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAutre">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labType">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listPanier">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Demander</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
